database/migrations/2023_11_21_163804_create_airlines_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('airlines', function (Blueprint $table) {
+            $table->id();
+            $table->string('name', 255);
```

Comment:
same here, name should be unique as it's the only way of differentiate one airline from another. It's a requirement for you to check that an airline is not repeated.